lighthouse-core/config/index.js

```diff
@@ -171,16 +172,79 @@ function filterAudits(audits, auditWhitelist) {
   return filteredAudits;
 }
 
-function expandAudits(audits) {
+function expandAudits(audits, paths) {
+  const rootPath = path.join(__dirname, '../../');
```

Comment:
That way if our auditList is fully hydrated, then we skip looking in these other paths.